src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -443,40 +443,6 @@ default CommandEditAction editCommandById(long id)
     @CheckReturnValue
     RestAction<Map<String, List<CommandPrivilege>>> updateCommandPrivileges(@Nonnull Map<String, Collection<? extends CommandPrivilege>> privileges);
 
-    /**
-     * Retrieves the available regions for this Guild
-     * <br>Shortcut for {@link #retrieveRegions(boolean) retrieveRegions(true)}
-     * <br>This will include deprecated voice regions by default.
-     *
-     * @return {@link net.dv8tion.jda.api.requests.RestAction RestAction} - Type {@link java.util.EnumSet EnumSet}
-     *
-     * @deprecated Guilds no longer have the {@link net.dv8tion.jda.api.Region Region} option. Use {@link VoiceChannel#getRegion()} instead.
-     */
-    @Nonnull
-    @CheckReturnValue
-    @Deprecated
-    @ForRemoval(deadline = "5.0.0")
-    default RestAction<EnumSet<Region>> retrieveRegions()
-    {
-        return retrieveRegions(true);
-    }
-
-    /**
-     * Retrieves the available regions for this Guild
-     *
-     * @param  includeDeprecated
-     *         Whether to include deprecated regions
-     *
-     * @return {@link net.dv8tion.jda.api.requests.RestAction RestAction} - Type {@link java.util.EnumSet EnumSet}
-     *
-     * @deprecated Guilds no longer have the {@link net.dv8tion.jda.api.Region Region} option. Use {@link VoiceChannel#getRegion()} instead.
-     */
-    @Nonnull
-    @CheckReturnValue
-    @Deprecated
-    @ForRemoval(deadline = "5.0.0")
-    RestAction<EnumSet<Region>> retrieveRegions(boolean includeDeprecated);
```

Comment:
I think this endpoint should still be supported, not sure why it was deprecated?